conf/distro/nilrt.inc

```diff
@@ -147,5 +151,6 @@ IPK_FEED_URIS += "\
     NIOE-all##${NILRT_LOCAL_FEED_URI}/all \
     NIOE-${MACHINE}##${NILRT_LOCAL_FEED_URI}/${MACHINE} \
     NIOE-${TUNE_PKGARCH}##${NILRT_LOCAL_FEED_URI}/${TUNE_PKGARCH} \
-    NI-software##${NILRT_MACHINE_FEED_URI_nimain} \
+    NI-main-software##${NILRT_MACHINE_FEED_URI_nimain} \
+    NI-lv-software##${NILRT_MACHINE_FEED_URI_lv2020} \
```

Comment:
I was talking about these lines actually. I can't remember now whether or not these sources will only be used during do_rootfs, or if OE will actually write them into the permanent `/etc/opkg/base-feeds.conf` (or whatever the file is called.)

If it does write them, then I think they will interfere with the source files which the SystemImage installer adds after us. And also, I'm not sure how the MAX workflows are going to react, given that we [intentionally chose](https://review-board.natinst.com/r/327590/diff/1#index_header) the names of those source feeds, with approval from HW config.